portfolio/src/main/webapp/style.css

```diff
@@ -36,4 +46,9 @@ img {
 
 .topnav a.active {
   border-bottom: 3px solid red;
-}
\ No newline at end of file
+}
+
+video {
```

Comment:
you have a video?! 😲

Reply:
Yeah! It's a two of my friends made as a recap of sophomore year (prior to covid)